lib/node/failure_detector.ml

```diff
@@ -1,6 +1,7 @@
 open Common
 open Common.Util
 open Types
+open Bin_prot.Common
```

Comment:
Kind of confused; you opened this module but I don't see any changes/additions that make use of opening it?

Reply:
Removed, was a wrong tentative on working on nix/macos branch.